packages/components/forms/src/text-input/TextInput.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+const getStyles = () => ({
+  inputWithCopyButton: css({
+    borderRadius: `${tokens.borderRadiusMedium} 0 0 ${tokens.borderRadiusMedium}`,
+  }),
+  copyButton: css({
+    button: {
+      borderLeft: 'none',
+      height: '100%',
+      borderRadius: `0 ${tokens.borderRadiusMedium} ${tokens.borderRadiusMedium} 0`,
+
+      '&:focus': {
```

Comment:
most likely those styles will be moved to CopyButton (we would also adjust button secondary in that case) - it is discussed with designers